### ISSUE TITLE
Pagination bootstrap display error

### ISSUE THREAD

**Issue by nguyentranchung** [livewire/livewire#1563]
### Description
![image](https://user-images.githubusercontent.com/9611224/92997523-eff0a080-f54e-11ea-9317-926533cc2235.png)

**Context**
- Livewire version: 2.1
- Laravel version: 8.x
- Browser: Chrome

**Comment by nguyentranchung** [comment]
Bootstrap sample
```html
<nav aria-label="Page navigation example">
  <ul class="pagination">
    <li class="page-item"><a class="page-link" href="#">Previous</a></li>
    <li class="page-item"><a class="page-link" href="#">1</a></li>
    <li class="page-item"><a class="page-link" href="#">2</a></li>
    <li class="page-item"><a class="page-link" href="#">3</a></li>
    <li class="page-item"><a class="page-link" href="#">Next</a></li>
  </ul>
</nav>
```
Livewire render
```html
<nav>
    <ul class="pagination">
        <span>
            <li class="page-item"><button type="button" dusk="previousPage" class="page-link" wire:click="previousPage" rel="prev" aria-label="« Previous">‹</button></li>
        </span>
        <span>
            <li class="page-item" wire:key="paginator-page-1"><button type="button" class="page-link" wire:click="gotoPage(1)">1</button> </li>
            <li class="page-item" wire:key="paginator-page-2"><button type="button" class="page-link" wire:click="gotoPage(2)">2</button> </li>
            <li class="page-item disabled" aria-disabled="true"><span class="page-link">...</span></li>
            <li class="page-item" wire:key="paginator-page-26"><button type="button" class="page-link" wire:click="gotoPage(26)">26</button> </li>
            <li class="page-item" wire:key="paginator-page-27"><button type="button" class="page-link" wire:click="gotoPage(27)">27</button> </li>
            <li class="page-item" wire:key="paginator-page-28"><button type="button" class="page-link" wire:click="gotoPage(28)">28</button> </li>
            <li class="page-item" wire:key="paginator-page-29"><button type="button" class="page-link" wire:click="gotoPage(29)">29</button> </li>
            <li class="page-item" wire:key="paginator-page-30"><button type="button" class="page-link" wire:click="gotoPage(30)">30</button> </li>
            <li class="page-item" wire:key="paginator-page-31"><button type="button" class="page-link" wire:click="gotoPage(31)">31</button> </li>
            <li class="page-item active" wire:key="paginator-page-32" aria-current="page"><span class="page-link">32</span></li>
            <li class="page-item" wire:key="paginator-page-33"><button type="button" class="page-link" wire:click="gotoPage(33)">33</button> </li>
            <li class="page-item" wire:key="paginator-page-34"><button type="button" class="page-link" wire:click="gotoPage(34)">34</button> </li>
            <li class="page-item" wire:key="paginator-page-35"><button type="button" class="page-link" wire:click="gotoPage(35)">35</button> </li>
        </span>
        <span>
            <li class="page-item"><button type="button" dusk="nextPage" class="page-link" wire:click="nextPage" rel="next" aria-label="Next »">›</button> </li>
        </span>
    </ul>
</nav>
```

**Comment by calebporzio** [comment]
Ok, this is fixed in v2.2.2
